components/automate-ui/src/app/modules/infra-proxy/create-infra-role-modal/create-infra-role-modal.component.ts

```diff
@@ -282,4 +247,7 @@ export class CreateInfraRoleModalComponent implements OnInit, OnDestroy {
     this.overrideTab = false;
   }
 
+  private isNavigationKey(event: KeyboardEvent): boolean {
+    return event.key === 'Shift' || event.key === 'Tab';
+  }
 }
```

Comment:
We have a utilities file that contains this function now, so you can now import it from `/automate-ui/src/app/helpers/utilities/utilities.ts`

Reply:
updated the changes